src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -1488,19 +1482,19 @@ public PropertyDescriptor getColumn(Domain domain, String columnName)
         List<Map<String, Object>> validators = (ArrayList<Map<String, Object>>)column.getAllProperties().get("propertyValidators");
         Map<String, Object> validator = validators.stream()
                 .filter(a-> a.get("name").equals(name))
-                .collect(Collectors.toList())
-                .get(0);
+                .findFirst().orElse(null);
+        assertNotNull("did not find expected validator ["+name+"] on column", validator);
```

Comment:
Include properties in failure message.
```suggestion
        assertNotNull("did not find property validator ["+name+"] on column. Column properties: " + column.getAllProperties().toString(), validator);
```